portfolio/src/main/webapp/index.html

```diff
@@ -40,7 +53,6 @@ <h1 style= "color:red;"> All about Taniece </h1>
         to Manager or even Director. Also, I aspire to travel the world.
         </b>
         </p>
-    </div>
-    <div id="comments-container"> </div>
+        </div>
```

Comment:
I don't see a corresponding '\<div\>' for this one.

Reply:
line 28